client/src/registerServiceWorker.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-console */
```

Comment:
no strong feelings one way or the other, I don't see much harm in keeping it, may as well just comment you the 'registerServiceWorker()' line from index and keep the file in case we'd like to re-enable